src/make_xlsx.py

```diff
@@ -322,29 +322,37 @@ def gen_instance_count_history_chart(workbook, header_format, val_format):
 
 
 def gen_instance_size_recommendations(workbook, header_format, val_format):
+    def transform(h, v):
+        if h == "cpu_usage":
+            return "%.2f%%" % (float(v)*100)
+        return v
+
     with utils.csv_folder(IN_INSTANCE_SIZE_RECOMMENDATIONS_DIR) as source:
         worksheet = workbook.add_worksheet("Instance size recommendations")
 
-        worksheet.set_column("A:F", 25)
-        worksheet.set_column("G:G", 40)
-        worksheet.merge_range("A1:E1", "Instance", header_format)
-        worksheet.merge_range("F1:F2", "Recommended", header_format)
-        worksheet.merge_range("G1:G2", "Reason", header_format)
+        worksheet.set_column("A:E", 25)
+        worksheet.set_column("F:F", 20)
+        worksheet.set_column("G:G", 18)
+        worksheet.set_column("H:H", 35)
+        worksheet.merge_range("A1:F1", "Instance", header_format)
+        worksheet.merge_range("G1:G2", "Recommended", header_format)
+        worksheet.merge_range("H1:H2", "Reason", header_format)
 
         refs = {
             "account": [0, "Account"],
             "id": [1, "ID"],
             "name": [2, "Name"],
             "size": [3, "Type"],
             "lifecycle": [4, "Lifecycle"],
-            "recommendation": [5, "Recommendation"],
-            "reason": [6, "Reason"]
+            "cpu_usage": [5, "CPU Utilization (Avg.)"],
+            "recommendation": [6, "Recommendation"],
+            "reason": [7, "Reason"]
         }
         for i in refs.values():
             worksheet.write(1, i[0], i[1], header_format)
             for i, line in zip(itertools.count(2), source):
                 for h, v in line.items():
-                    worksheet.write(i, refs[h][0], v, val_format)
+                    worksheet.write(i, refs[h][0], transform(h, v), val_format)
```

Comment:
I had a crash when testing this PR with the trackit account:

Generating xlsx file.
Traceback (most recent call last):
  File "src/make_xlsx.py", line 511, in <module>
    main(name)
  File "src/make_xlsx.py", line 498, in main
    gen_instance_size_recommendations(workbook, header_format, val_format)
  File "src/make_xlsx.py", line 355, in gen_instance_size_recommendations
    worksheet.write(i, refs[h][0], transform(h, v), val_format)
  File "src/make_xlsx.py", line 327, in transform
    return "%.2f%%" % (float(v)*100)
ValueError: could not convert string to float: